src/geometry/WorkingGeometry.ts

```diff
@@ -168,20 +211,13 @@ export class WorkingGeometry {
      * @param {vec3} pullPoint: point that will be scaled by factor away from holdPoint.
      * @param {vec3} holdPoint: point to scale from, default to true origin.
      */
-    public scaleByFactor(factor: number, pullPoint: vec3, holdPoint: vec3 = vec3.create()) {
-        const scalingDirection: vec3 = vec3.sub(vec3.create(), pullPoint, holdPoint);
-        const absScalingDirection: vec3 = vec3.fromValues(
-            Math.abs(scalingDirection[0]),
-            Math.abs(scalingDirection[1]),
-            Math.abs(scalingDirection[2])
-        );
+    public scaleByFactor(factor: number, holdPoint: vec3 = vec3.create()) {
```

Comment:
Why is there no longer a `pullPoint`? The shape can be scaled by a factor but it has to scale to a certain direction, unless we want to define it as always scaling to the positive quadrant and then move it later if needed.

Reply:
I removed it because I misread what was going on here before as using the distance as part of the scaling factor, which can produce incorrect results like zero-scaling along some axes. However, if we want this as a "rigid body" counterpart to the modified scale function, it would make sense to add that back, so I'll do that.

Reply:
Ah never mind, I think adding back `pullPoint` still isn't necessary because this is uniform scaling, so it scales equally in all directions — we don't need to specify a specific direction. The only thing that needs to be specified is where the scale originates from, which is `holdPoint`. We might want to add a rotation here like in the other function, but then we would also need to specify a `destinationPoint` to determine the rotation direction. Since the scaling factor is specified directly, that `destinationPoint` might not actually be the real destination point, so if we decide to add a `destinationPoint`, we should remove `factor`.

So basically I think there are two options here:
1) Leave the interface as is. A user can't specify a rotation directly in this function, but it's easy to understand and use.
2) Use the same interface as the other function. This makes it more complex, but then the difference between the two functions is more clear.

I'm personally leaning towards option 1 (and perhaps using a similar interface for the other function as well, but I'm not sure if that would make it easier to use, so I won't do it in this PR regardless). Thoughts?